test/benchmarks/build.sbt

```diff
@@ -0,0 +1,11 @@
+scalaHome := Some(file("../../build/pack"))
+scalaVersion := "2.11.8"
```

Comment:
I suppose this should be "2.12.0" (not sure whether it matters if `scalaHome` is set)